p/psad/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Base Image
+FROM  registry.access.redhat.com/ubi8/ubi:8.3
+# The author
+MAINTAINER "nthorat"
+
+ENV PSAD_VERSION 3.0
+ENV PSAD_REPOSITORY https://github.com/mrash/psad
+ENV PSAD_DIR psad
+ENV PERL_MM_USE_DEFAULT 1 
+
+# Install dependent packages
+RUN cd $HOME\
+    && yum install -y git wget vim\
+    && yum install -y make\
```

Comment:
can we use a single "yum install" command?

Reply:
Done.